packages/cli/bin/index.js

```diff
@@ -10,13 +10,16 @@ const start = require('../lib/start');
 const devPlugin = require('../lib/devPlugin');
 const dataset = require('../lib/dataset');
 const childProcess = require('child_process');
+const debugLog = require('debug')('cli/bin');
 
 // check node version
 if (!semver.gte(process.version, '10.0.0')) {
-  console.log(
+  debugLog(
     chalk.red(
-      `Pipcook requires node version higher than node 10.x. Howeverm your kicak node version is ${process.version}, ` +
-      'Please update node.js'
+      `Pipcook requires node version higher than node 10.x. 
+      Howeverm your kicak node version is 
+      ${process.version}
+      , Please update node.js`
     )
```

Comment:
Debugging should not cover user-land logs, here it's used to tell users which node is older, so it's good to keep using log.